drivers/tapdisk-nbdserver.h

```diff
@@ -98,5 +100,28 @@ int tapdisk_nbdserver_listen_unix(td_nbdserver_t *server);
 void tapdisk_nbdserver_free(td_nbdserver_t *);
 void tapdisk_nbdserver_pause(td_nbdserver_t *);
 int tapdisk_nbdserver_unpause(td_nbdserver_t *);
+void tapdisk_nbdserver_clientcb(event_id_t id, char mode, void *data);
+int tapdisk_nbdserver_reqs_init(td_nbdserver_client_t *client, int n_reqs);
+void tapdisk_nbdserver_free_client(td_nbdserver_client_t *client);
+td_nbdserver_client_t *tapdisk_nbdserver_alloc_client(td_nbdserver_t *server);
+bool tapdisk_nbdserver_contains_client(td_nbdserver_t *server,
```

Comment:
Given you added a description for a trivial function like tapdisk_nbdserver_reqs_pending
why not adding one here? ;-)